checkov/common/bridgecrew/wrapper.py

```diff
@@ -101,6 +104,25 @@ def persist_run_metadata(
     object_path = f'{full_repo_object_key}/{checkov_results_prefix}/run_metadata.json'
     try:
         s3_client.put_object(Bucket=bucket, Key=object_path, Body=json.dumps(run_metadata, indent=2))
+
+    except Exception:
+        logging.error(f"failed to persist run metadata into S3 bucket {bucket}", exc_info=True)
+        raise
+
+
+def persist_logs_stream(logs_stream: StringIO, s3_client: BaseClient, bucket: str, full_repo_object_key: str) -> None:
```

Comment:
@gruebel The tar compression works successfully.

Reply:
@reemodo I'm highly recommend:
1. moving the compression logic to here: `checkov/common/util/file_utils.py`
A suggestion
2. creating UTs for the compression part + making sure you can decompress it to the original data: